src/Resque/Component/Core/Process.php

```diff
@@ -123,7 +123,7 @@ public function wait()
             );
         }
 
-        pcntl_waitpid($this->pid, $this->status);
+        $status = pcntl_waitpid($this->pid, $this->status, $options);
```

Comment:
I'm happy for wait to return $status instead of $this. Thoughts? I'm not a massive fan of referenced variables, and returning $this just because is probably a bad idea anyway.